ipysheet/renderer_nbext.py

```diff
@@ -1,8 +0,0 @@
-
-def _jupyter_nbextension_paths():
```

Comment:
I removed this file, this function was duplicated in `__init__.py`. Hope it's fine

Reply:
It's not the same extension, ipysheet has 2.

Reply:
Ok :) Will put it back then. Thanks for the quick review!